README.md

```
@@ -464,6 +466,82 @@ This reverse proxy will work with any application that supports qBittorrent's We
+
+#### How It Works
+
+1. autobrr sees a new release from a tracker
+2. autobrr sends the torrentname and target instance ID to qui's webhook endpoint
+3. qui searches the specified qBittorrent instance for matching content
+4. qui responds whether you can cross-seed (200 OK) or not (404 Not Found)
+5. autobrr can decide to download based on the response
+
+#### Setup
+
+**1. Create an API Key in qui**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)